dbms/tests/performance/linear_regression.xml

```diff
@@ -0,0 +1,40 @@
+<test>
+    <name>linear_regression</name>
+    <type>loop</type>
+
+    <stop_conditions>
+        <any_of>
+            <average_speed_not_changing_for_ms>3000</average_speed_not_changing_for_ms>
+            <total_time_ms>5000</total_time_ms>
+        </any_of>
+    </stop_conditions>
+
+    <main_metric>
+        <min_time/>
+    </main_metric>
+
+    <create_query>CREATE TABLE train_dataset(p1 Float64, p2 Float64, target Float64) ENGINE = Memory</create_query>
+    <create_query>CREATE TABLE test_dataset(p1 Float64, p2 Float64) ENGINE = Memory</create_query>
+
+    <create_query>CREATE TABLE test_model engine = Memory as select linearRegressionState(0.001)(target, p1, p2) as state from train_dataset</create_query>
+
+    <fill_query>INSERT INTO train_dataset values (2.0, 1.0, 6.0), (3.0, -1.0, 6.0), (4.0, 2.0, 11.0), (-2.0, 1.0, -2.0)</fill_query>
```

Comment:
Table is way too small. Better to reuse Taxi, Ontime or Metrica dataset
(look at the used tables across other performance tests).

There are many useful queries like "predict ride price according to time, distance and number of passengers".